tests/unittest_builder.py

```diff
@@ -790,5 +797,53 @@ def test_parse_module_with_invalid_type_comments_does_not_crash():
     assert isinstance(node, nodes.Module)
 
 
+def test_c_module_text_signature():
```

Comment:
I think this test cannot pass on pypy, since they do not support CPython extensions natively.

Reply:
We should be able to tell pytest to skip it right?

Although I must confess I don't know if this is actually true. Did you find any documentation about this?

Reply:
> We should be able to tell pytest to skip it right?

Yes we can

> Although I must confess I don't know if this is actually true. Did you find any documentation about this?

Yes I've read that from pypy's documentation, but I can't find the link now. But here's what the [pypy wiki](https://en.wikipedia.org/wiki/PyPy) says: "There is a compatibility layer for CPython C API extensions called CPyExt, but it is incomplete and experimental.". So I'm positive that, by default, pypy has no support for c-modules built for cpython.

Reply:
Let's skip then!